corehq/util/jsonattrs.py

```diff
@@ -209,6 +219,27 @@ def jsonify(self, value):
         return {k: to_json(v) for k, v in value.items()}
 
 
+@define
+class AttrsObjectBuilder:
```

Comment:
Is it the case that an `AttrsList` is a list of objects, and an `AttrsDict` is a dict whose values are objects, whereas this represents a single object?  There are some repeated patterns between the three - would be nice to combine the builders reflecting the commonalities if that's doable.

Reply:
> Is it the case that an AttrsList is a list of objects, and an AttrsDict is a dict whose values are objects, whereas this represents a single object?

Yes, that's correct.

> There are some repeated patterns between the three - would be nice to combine the builders reflecting the commonalities if that's doable.

I'll take a look